velox/common/compression/tests/CompressionTest.cpp

```diff
@@ -122,6 +122,19 @@ void checkCodecRoundtrip(
   decompressed.resize(data.size());
   ASSERT_EQ(data, decompressed);
   ASSERT_EQ(data.size(), decompressedLength);
+
+  // Decompress corrupted data.
+  std::vector<uint8_t> corruptedData = compressed;
+  corruptedData.resize(compressed.size() + 1);
+  VELOX_ASSERT_ERROR_STATUS(
+      c2->decompress(
+            corruptedData.data(),
+            corruptedData.size(),
+            decompressed.data(),
+            decompressed.size())
+          .error(),
+      StatusCode::kIOError,
+      "decompression failed");
```

Comment:
Please include the `ret` zstdError("ZSTD decompression failed: ", ret), if it is different with codec, please add an example in the comment

Reply:
Sorry for missing these comments previously. While addressing this comment, I found the error message for the lz4 compression was incomplete. So I extracted this part of change to another pr. Could you help to review that one first? Thanks! 

Here's the pr link: https://github.com/facebookincubator/velox/pull/14570